src/routes/home/Home.js

```diff
@@ -431,6 +458,34 @@ class Home extends React.Component {
       formatted_address: 'Finding Address...',
       addressProvenance,
     });
+
+    const {
+      state: { boroughBoundariesFeatureCollection },
+    } = this;
+    // TODO the above/below blocks were copied/adapted from ElectriCitibikes.js, consider unifying them
+    if (boroughBoundariesFeatureCollection) {
+      console.time('new PolygonLookup'); // eslint-disable-line no-console
+      const lookup = new PolygonLookup(boroughBoundariesFeatureCollection);
+      console.timeEnd('new PolygonLookup'); // eslint-disable-line no-console
+
+      const end = { latitude, longitude };
+      const BoroName = getBoroNameMemoized({ lookup, end });
+      if (BoroName === '(unknown borough)') {
+        const errorMessage = 'Please select a location within NYC';
+        this.setState({
+          formatted_address: errorMessage,
+          coordsAreInNyc: false,
+        });
+        this.notifyError(
+          `latitude/longitude (${latitude}, ${longitude}) is outside NYC. ${errorMessage}`,
+        );
```

Comment:
when using the map to select a location, the error toast appears behind the map as soon as the location is chosen, rather than when the map is closed. For example, type `90210` into the search box and click the first result. We can fix this by either:

* making the toast appear above the map (probably some z-index tweaking)
* re-evaluating the location when them map closes (not as good IMO)

Reply:
> making the toast appear above the map (probably some z-index tweaking)

fixed in 456907dca438280beb63fd65481ed6e1dc6b2f8b